Mastodon/Scene/Profile/Header/View/ProfileHeaderView.swift

```diff
@@ -172,15 +172,26 @@ final class ProfileHeaderView: UIView {
         textField.autocapitalizationType = .none
         return textField
     }()
-
-    let usernameLabel: UILabel = {
-        let label = UILabel()
-        label.font = UIFontMetrics(forTextStyle: .callout).scaledFont(for: .systemFont(ofSize: 16, weight: .regular))
-        label.adjustsFontSizeToFitWidth = true
-        label.minimumScaleFactor = 0.5
-        label.textColor = Asset.Colors.Label.secondary.color
-        label.text = "@alice"
-        return label
+    
+    private lazy var usernameButtonMenu: UIMenu = {
+        UIMenu(children: [
+            UIAction(title: L10n.Common.Controls.Actions.copy, handler: { [weak self] _ in
```

Comment:
Two thoughts:

1. Should this use the [new API for edit menus](https://developer.apple.com/documentation/uikit/uieditmenuinteraction/) in iOS 16? (so that users get the little mini popover thing like when you select text)
2. Can you add a `doc.on.doc` icon so the menu doesn’t have the text alone?

Reply:
I disagree with 1., as we still have to support iOS 15 and another `@available` with two different implementations is not worth the effort in my opinion. But I like 2.!

Reply:
Very good point @j-f1 , added `doc.on.doc`.